lib/src/widgets/form_control_inherited_notifier.dart

```diff
@@ -13,13 +13,13 @@ import 'package:reactive_forms/src/widgets/inherited_streamer.dart';
 ///
 /// The [Listenable] is provided by the [notifierDelegate].
 ///
-class FormControlInheritedStreamer extends InheritedStreamer<dynamic> {
+class FormControlInheritedStreamer extends InheritedStreamer<Object?> {
```

Comment:
Hi @kuhnroyal, Have you tested this change of data type? Are you sure it will not launch a cast exception at some point?

Reply:
Well, `Object?` means it can be cast to any subclass of `Object` to `null` or as always, to `dynamic`. There is nothing else left :)
`Object?` is the new `dynamic` in the post null-safety world. And since I changed ALL occurrences of dynamic it should be safe.
And all tests succeed, if there is a problem somewhere, then it needs a test and should solved at the source.